src/php/binaries.rs

```diff
@@ -29,10 +29,9 @@ pub(crate) fn all() -> HashMap<PhpVersion, PhpBinary> {
 
     binaries_from_env(&mut binaries);
 
-    merge_binaries(
-        binaries_from_dir(PathBuf::from("/usr/bin")),
-        &mut binaries
-    );
+    merge_binaries(binaries_from_dir(PathBuf::from("/usr/bin")), &mut binaries);
+    merge_binaries(binaries_from_dir(PathBuf::from("/usr/sbin")), &mut binaries);
+    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/")), &mut binaries);
```

Comment:
Actually that didn't change anything, I guess mostly because you missed the `/bin` at the end.
I tried locally, and that shows the same version like Symfony:
```suggestion
    if cfg!(target_os = "macos") {
        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/bin")), &mut binaries);
        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php@*/*/bin")), &mut binaries);
        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/php*/bin")), &mut binaries);
    }
```
with a big **BUT** for some reasons all list entries are empty, except the version:
```
┌─────────┬─────────────────────────────────────┬──────────────────────────────────────────┬─────────┐
| Version | PHP CLI                             | PHP FPM                                  | PHP CGI |
├─────────┼─────────────────────────────────────┼──────────────────────────────────────────┼─────────┤
| 7.3.11  | /usr/bin/php                        | /usr/sbin/php-fpm                        |         |
| 7.3.21  |                                     |                                          |         |
| 7.4.9   | /usr/local/Cellar/php/7.4.9/bin/php | /usr/local/Cellar/php/7.4.9/sbin/php-fpm |         |
| 5.5.5   |                                     |                                          |         |
└─────────┴─────────────────────────────────────┴──────────────────────────────────────────┴─────────┘
```
I will push a PR

Reply:
PR: https://github.com/Pierstoval/rymfony/pull/17/files